src/charactersheet/viewmodels/common/player_card/index.html

```diff
@@ -16,13 +19,16 @@
         <div class="col-xs-12 text-center">
           <div class="h4">
             <span data-bind="text: player.name"></span><br />
-            <small data-bind="text: player.playerSummary"></small>
+            <small data-bind="text: player.summary"></small>
           </div>
+          <small class="text-warning" data-bind="visible: player.isBloodied">
+            <span data-bind="text: player.name"></span> is bloodied!
+          </small>&nbsp;
         </div>
       </div>
       <!-- Health, Spells, and Features  -->
       <div class="row">
-        <div class="col-xs-12">
+        <div class="col-xs-12" data-bind="visible: player.hitPoints">
```

Comment:
I think this is somehow hiding the HP value for me. I can't see my player's HP anymore.
![image](https://user-images.githubusercontent.com/7286387/147905713-4b82d5b3-f1dc-45ed-aba3-3fd8f81215a6.png)

Reply:
Oops. Yeah I use the same card for DMs and players. Will fix.